chargebee/src/main/java/com/chargebee/android/billingservice/CBPurchase.kt

```diff
@@ -13,12 +13,18 @@ import com.chargebee.android.models.ResultHandler
 import com.chargebee.android.network.*
 import com.chargebee.android.resources.CatalogVersion
 import com.chargebee.android.resources.ReceiptResource
-import java.util.ArrayList
+
 object CBPurchase {
 
     var billingClientManager: BillingClientManager? = null
     val productIdList = arrayListOf<String>()
     private var customer : CBCustomer? = null
+    var inActivePurchases = false
```

Comment:
This should be `private`.

Reply:
made it as internal as its used in other class.